driver.go

```diff
@@ -204,7 +216,7 @@ func (c zConn) QueryContext(ctx context.Context, query string, args []driver.Nam
 
 func (c zConn) Prepare(query string) (stmt driver.Stmt, err error) {
 	if c.options.AllowRootSpan {
-		span := c.tracer.StartSpan("sql/prepare", zipkin.Kind(zipkinmodel.Client))
+		span := c.tracer.StartSpan("sql/prepare")
```

Comment:
Personally I would not remove the zipkin.Kind here yet and defer that to the "driver" specific profiles you want to introduce. In a fully fledged DB this action will call the backend.